scripts/publishing-common.sh

```diff
@@ -13,18 +13,20 @@
 # License for the specific language governing permissions and
 # limitations under the License.
 
-export VERSION=$(cat $(dirname "${0}")/../VERSION)
+DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
```

Comment:
This path resolution method should be used at [the beginning of the script as it sources the helpers](https://github.com/aws/amazon-ecs-agent/pull/1957/files#diff-4ccd150869ada089e3677e1655559c73R25), though the use of this script by `source`-ing is a bit less tempting with these latest changes so its not as critical to block on. As with our earlier conversation about sourcing and considerations - keep in mind this could be clobbering a consumer's variable.

If its not going to get fixed now, let's jot down some notes (can you make an issue?) on what to change so that it gets fixed up at some point.